edc-tests/pom.xml

```
@@ -0,0 +1,122 @@
+        <dependency>
+            <groupId>net.catenax.edc</groupId>
+            <artifactId>edc-controlplane-postgresql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>net.catenax.edc</groupId>
+            <artifactId>edc-controlplane-postgresql-hashicorp-vault</artifactId>
+        </dependency>
+
+        <!-- edc-dataplane -->
+        <dependency>
+            <groupId>net.catenax.edc</groupId>
+            <artifactId>edc-dataplane-azure-vault</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>net.catenax.edc</groupId>
+            <artifactId>edc-dataplane-hashicorp-vault</artifactId>
+        </dependency>
+    </dependencies>
+
+    <properties>
+        <jacoco.outputDirectory>${project.reporting.outputDirectory}/jacoco-aggregate</jacoco.outputDirectory>
+        <sonar.coverage.jacoco.xmlReportPaths>${jacoco.outputDirectory}/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+        <sonar.organization>catenax-ng</sonar.organization>
+        <sonar.projectKey>${sonar.organization}_product-edc</sonar.projectKey>
+    </properties>
+
+    <profiles>
+        <profile>
+            <id>coverage</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.jacoco</groupId>
+                        <artifactId>jacoco-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>report-aggregate</id>
+                                <phase>verify</phase>
+                                <goals>
+                                    <goal>report-aggregate</goal>
+                                </goals>
+                                <configuration>
+                                    <outputDirectory>${jacoco.outputDirectory}</outputDirectory>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>sonar</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.sonarsource.scanner.maven</groupId>
+                        <artifactId>sonar-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>sonar</id>
+                                <phase>verify</phase>
+                                <goals>
+                                    <goal>sonar</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+</project>
```

Missing empty line at EOF
```suggestion
</project>

```